prover/src/lib.rs

```diff
@@ -111,6 +111,7 @@ impl Prover for ExecutionProver {
     type Air = ProcessorAir;
     type BaseField = Felt;
     type Trace = ExecutionTrace;
+    type HashFn = Rpo256;
```

Comment:
This should still be `BLAKE3` (coming from `miden-crypto`).

We will also need to make this generic to accept different hash functions - but this could be done in a subsequent PR.

Reply:
Will handle on the previous item issue